lessons/ro/chapter_10.yaml

```
@@ -0,0 +1,145 @@
+    - `./main > output.txt`
+    - `./main >> output.txt`
+    - `./main 2> output.txt`
+    - `./main 2>> output.txt`
+    
+    > Comenzile cu `2` vor copia doar erorile generate de program, iar cele fără `2` vor elimina toate erorile.
+
+    > Comanda cu `>>` va adăuga text la sfârșitul fișierului, în timp ce operatorul `>` va suprascrie fișierul.
+  code: >-
+    https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=use+std%3A%3Aio%3B%0A%0Afn+main%28%29+%7B%0A++++let+mut+input+%3D+String%3A%3Anew%28%29%3B%0A%0A++++%2F%2F+citirea+va+fi+oprita+la+apri%C8%9Bia+caracterului+%60%5Cn%60%0A++++if+let+Ok%28_%29+%3D+io%3A%3Astdin%28%29.read_line%28%26mut+input%29+%7B%0A++++++++println%21%28%22Textul+de+intrare+%3A+%7B%7D%22%2C+input%29%3B%0A++++%7D%0A%7D%0A
+
+- tile: Descriptori de Fișiere
+  content_markdown: >
+    Acum că știm care sunt operațiunile de bază `I/O`, intrăm mai în detaliu.
+
+    Ai mai văzut deja: `stdin` (intrare standard), `stdout` (ieșire standard) și `stderr` (eroare standard).
+    Pentru fiecare dintre ele i se asociază un număr întreg pozitiv, un identificator unic
+    pentru un canal `I/O` (de exemplu: fișier), cunoscut sub numele de `descriptor de fișier (fd)`.
+
+    Prin urmare:
+    - `stdin`: 0
+    - `stdout`: 1
+    - `stderr`: 2
+    - fișiere
+
+    [![file descriptors](https://cs-pub-ro.github.io/operating-systems/assets/images/file-descriptors-d19424f0a417ecd1c032f98a1969ad75.svg)
+
+    Lucrul cu fișiere joacă un rol important în operațiunile `I/O`.
+
+    Ați învățat deja să deschideți un fișier text și să citiți conținutul acestuia.
+
+    Dar ce zici de a scrie date în el, ca un canal `I/O`.
+  code: >-
+    https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=use+std%3A%3Afs%3A%3A%7Bself%2C+File%7D%3B++++++%2F%2F+sistemul+de+fi%C5%9Fiere%0Ause+std%3A%3Aio%3A%3A%7Bself%2C+Write%7D%3B+++++%2F%2F+input+output%0A%0Afn+main%28%29+-%3E+io%3A%3AResult%3C%28%29%3E+%7B%0A++++let+nume_fi%C5%9Fier+%3D+%22output.txt%22%3B%0A%0A++++%2F%2F+creaz%C4%83+un+fi%C5%9Fier+dac%C4%83+nu+exist%C4%83+deja%0A++++let+mut+fi%C5%9Fier+%3D+File%3A%3Acreate%28nume_fi%C5%9Fier%29%3F%3B%0A%0A++++let+text_to_write+%3D+%22Buna%2C+lume%21%5Cn%5C%0A++++++++++++++++++++++++++++Aceasta+este+o+linie+de+text.%5Cn%22%3B%0A++++fi%C5%9Fier.write_all%28text_to_write.as_bytes%28%29%29%3F%3B%0A%0A++++let+absolute_path+%3D+fs%3A%3Acanonicalize%28nume_fi%C5%9Fier%29%3F%3B%0A++++println%21%28%22Textul+a+fost+scris+%C3%AE+fi%C5%9Fierul%3A+%7B%3A%3F%7D%22%2C+absolute_path%29%3B%0A%0A++++return+Ok%28%28%29%29%3B%0A%7D%0A
+- title: Argumentele din Linia de Comandă
+  content_markdown: >
+    Un program Rust este capabil să primească „input” din argumentele în linie.
+
+    Pentru a face acest lucru, deschideți un [terminal](https://www.youtube.com/watch?v=lZ7Kix9bjPI),
+    compilați-l și transmiteți argumentele executabilului din promptul de comandă.
```

```suggestion
    compilați programul și transmiteți argumentele executabilului din promptul de comandă.
```
Se intelege mai degraba compilati terminalul din context.